server/sonar-server/src/test/java/org/sonar/server/qualitygate/ws/CreateActionTest.java

```diff
@@ -104,14 +136,20 @@ public void test_ws_definition() {
     assertThat(action.params()).hasSize(2);
   }
 
```

Comment:
A test is missing when trying to create an quality gate having the same name as another one from the same organization => ko
And another one when trying to use a name alredy used, but in another org => ok